pallets/bridge-transfer/src/tests.rs

```diff
@@ -59,35 +59,30 @@ fn transfer() {
 			10,
 			resource_id,
 		));
-		assert_eq!(Balances::free_balance(&bridge_id), ENDOWED_BALANCE - 10);
 		assert_eq!(Balances::free_balance(RELAYER_A), ENDOWED_BALANCE + 10);
 
-		assert_events(vec![Event::Balances(balances::Event::Transfer {
-			from: Bridge::account_id(),
-			to: RELAYER_A,
+		assert_events(vec![Event::Balances(balances::Event::Deposit {
+			who: RELAYER_A,
 			amount: 10,
 		})]);
 	})
 }
 
 #[test]
-fn transfer_to_holdingaccount() {
+fn exceed_max_supply() {
```

Comment:
In this test we only checked the scenario where we transfer one-off balance that exceeds the limit, shall we have another test that verifies the behaviour when we have accumulated balance that exceeds the limit? E.g. the sum of 5 transfers exceeds the max supply.

Reply:
will add this case.